docs/developer/sound.md

```diff
@@ -0,0 +1,129 @@
+# Music and Sounds
+
+## Melodies
+
+The high level music API in arcade uses a syntax for creating melodies from
+sequences of notes. Melodies are authored using the `music.Melody` class
+in arcade which encodes them in a string syntax. For some example melodies,
+see the [mixer package](https://github.com/Microsoft/pxt-common-packages/blob/master/libs/mixer/melody.ts#L387).
+
```

Comment:
The high-level music API in arcade creates and processes melodies as a sequence of notes following a syntax form for each note. The melodies are authored using the `music.Melody` class in arcade which encodes them into a string with this syntax. For some examples of formatted melodies, see the [mixer extension/package](https://github.com/Microsoft/pxt-common-packages/blob/master/libs/mixer/melody.ts#L387).